ScienceJournal/UI/ObserveViewController.swift

```diff
@@ -898,10 +902,16 @@ open class ObserveViewController: ScienceJournalCollectionViewController, ChartC
 
   // Animates the display of the time axis view.
   func showTimeAxisView(_ isVisible: Bool) {
-    UIView.animate(withDuration: 0.5, animations: {
-      self.timeAxisController.timeAxisView.alpha = isVisible ? 1 : 0
-    }) { (_) in
-      self.adjustContentInsets()
+    if FeatureFlags.isActionAreaEnabled {
+      self.transitionCoordinator?.animate(alongsideTransition: { _ in
+        self.timeAxisController.timeAxisView.alpha = isVisible ? 1 : 0
+        self.adjustContentInsets()
+      }, completion: nil)
```

Comment:
I think `, completion: nil` can be omitted and might be removed by the swift formatter, if that gets set up. (The SwiftFormat config I've been using would remove it, but that's not official.)